components/python/import-profiler/Makefile

```diff
@@ -28,7 +28,7 @@ COMPONENT_FMRI=         library/python/import-profiler
 COMPONENT_CLASSIFICATION=Development/Python
 COMPONENT_LICENSE_FILE= $(COMPONENT_NAME).license
 COMPONENT_LICENSE=       MIT
-COMPONENT_SUMMARY=      "Import profiler to find bottlenecks in import times."
+COMPONENT_SUMMARY=      "Import_profiler_to_find_bottlenecks_in_import_times."
```

Comment:
I meant removing COMPONENT_SUMMARY from Makefile and adding it directly to manifest

Reply:
Hi @pyhalov 
I am not sure how to do it!

Looked around and found (for example) that COMPONENT_SUMMARY IS defined in Makefile for python/geoip and even with spaces (although w/o quotes)!
```
$ grep COMPONENT_SUMMARY ~/oi-userland/components/python/geoip/Makefile
COMPONENT_SUMMARY=      MaxMind GeoIP Legacy Database - Python API
```
Please advice.

Regards.

Reply:
Hi.
Remove COMPONENT_SUMMARY from Makefile and add directly to p5m file, like it's done, for example, at https://github.com/OpenIndiana/oi-userland/blob/3f4d054e2924db4c4e9bd68846dc4f8617d961e2/components/python/pycairo/pycairo-PYVER.p5m#L28 . 

geoip doesn't use versioned (PYV) manifests, so it's not affected.